packages/runtime/src/api/graphql/client.ts

```diff
@@ -0,0 +1,32 @@
+export class GraphQLClient {
+  private endpoint: string
+
+  constructor(endpoint: string) {
+    this.endpoint = endpoint
+  }
+
+  async request<TData, TVariables = Record<string, never>>(
```

Comment:
The generics aren't required so it's possible to miss them. If it's possible to make them required that would reduce the probability of making a mistake

Reply:
Not going to worry about this